packages/inventory-compliance/src/Compliance.js

```diff
@@ -28,12 +28,11 @@ query System($systemId: String!){
             rulesPassed
             lastScanned
             score
+            supported
+            ssgVersion
```

Comment:
@vkrizan @akofink Mind confirming that these are the right fields to request and show. (and that the benchmark query is not needed.) It should show the ssgVersion and supported of the reported policy.

Reply:
Yes, it looks right to me